pkg/sfu/videolayerselector.go

```diff
@@ -17,7 +17,7 @@ type targetLayer struct {
 type DDVideoLayerSelector struct {
 	logger logger.Logger
 
-	// TODO : fields for frame chain detect
+	// DD-TODO : fields for frame chain detect
```

Comment:
Marking all these as `DD-TODO` for easier grepping for dependency descriptor related changes.